metrics/influx/influx.go

```diff
@@ -0,0 +1,223 @@
+package metrics
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/prebid/prebid-cache/config"
+	"github.com/rcrowley/go-metrics"
+	"github.com/vrischmann/go-metrics-influxdb"
+)
+
+/**************************************************
```

Comment:
Consider removing the 'C' style comment blocks.

Reply:
Done